app/main.py

```diff
@@ -31,6 +31,7 @@
 
 # We start by instantiating the app (NB: did not try to look for other stylesheets yet)
 app = dash.Dash(__name__, external_stylesheets=[dbc.themes.UNITED])
+app.title = 'PyroNear'
```

Comment:
"Pyronear - Monitoring platform" to be more specific perhaps?

Reply:
sure, your call!